src/msalexample-ios-macos/Shared/MSALAuthentication.swift

```diff
@@ -0,0 +1,83 @@
+import MSAL
+
+class MSALAuthentication {
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    private static let kClientId = ""
+
+    // 'Tenant ID' of your Azure AD instance - this value is a GUID
+    private static let kTenantId = ""
+    
+    private static let kAuthority = try! MSALB2CAuthority(url: URL(string: "https://login.microsoftonline.com/\(kTenantId)")!)
+    private static let kConfig = MSALPublicClientApplicationConfig(clientId: kClientId, redirectUri: nil, authority: kAuthority)
+
+    // In order to take advantage of token caching, your MSAL client singleton must
+    // have a lifecycle that at least matches the lifecycle of the user's session in
+    // the app.
+    private static let kApplication: MSALPublicClientApplication = try! MSALPublicClientApplication(configuration: kConfig)
```

Comment:
Sorry I didn't pick up on this before, I think we've been calling them something more like...

`kMSALClient` so that when you see `kApplication` without the instantiation of this nearby (even more important now) you're not wondering what that is

Reply:
changed @ckittel

done | addressed from [f5765](https://github.com/Azure-Samples/ms-identity-docs-code-swift/pull/1/commits/f576578ca9c97f0a116329b48c0d6888d083f2b2)

Reply:
Looks good!